lib/engine/round/operating.rb

```diff
@@ -351,7 +366,7 @@ def start_operating
         @current_actions.clear
         log_operation(@current_entity)
         @current_entity.trains.each { |train| train.operated = false }
-        place_home_token(@current_entity) if @home_token_timing == :operate
+
```

Comment:
hmmmmmmm this is weird that this happens in a SKIP check

Reply:
Yeah, the other way would be to do a second check here to see if it needs laying